### PR TITLE
Fix encode_packed implementation

### PR DESCRIPTION
**Pull request by vovac12**

### CODE REVIEW

File: pallets/eth-bridge/src/requests/encode_packed.rs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            let mut arr = vec![0u8; size_bytes];
+            for i in 0..size_bytes {
+                arr[size_bytes - i - 1] = int.byte(i);
+            }
+            arr
+        }
+        RawToken(Token::Bool(b)) => {
+            vec![if b { 1 } else { 0 }]
+        }
+        // FIXME: the two cases below may work incorrectly on some inputs (e.g. multi-dimensional arrays).
```
Comment (N1ghtStorm) [inline pallets/eth-bridge/src/requests/encode_packed.rs:41]: I suggest to delete this comment

**Review by N1ghtStorm — APPROVED** [review]

File: pallets/eth-bridge/src/requests/encode_packed.rs (hunk 1 above)
Comment (wer1st) [reply pallets/eth-bridge/src/requests/encode_packed.rs:41]: Why?

**Review by wer1st — APPROVED** [review]

File: pallets/eth-bridge/src/tests/ethabi.rs [hunk 2]
```diff
@@ -0,0 +1,310 @@
+use common::DAI;
```
Comment (wer1st) [inline pallets/eth-bridge/src/tests/ethabi.rs:1]: Copyright

**Review by wer1st — APPROVED** [review]

File: pallets/eth-bridge/src/requests/encode_packed.rs (hunk 1 above)
Comment (vovac12) [reply pallets/eth-bridge/src/requests/encode_packed.rs]: Comment changed. It's better to not hide info about this

**Pull request merged**